org.scummvm.ScummVM.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id": "org.scummvm.ScummVM",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        /* ScummVM does not respect XDG base directories */
+        "--persist=.scummvm",
+        /* Allow Documents directory access - for storing ScummVM games etc. */
+        "--filesystem=xdg-documents"
+    ],
+    "command": "scummvm",
+    "rename-appdata-file": "scummvm.appdata.xml",
+    "rename-desktop-file": "scummvm.desktop",
+    "rename-icon": "scummvm",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/SDL/SDL-1.2.15.json",
+        {
+            "name": "scummvm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/scummvm/scummvm/archive/v2.0.0.tar.gz",
```

Comment:
Have you contacted upstream team if they would like to help with maintaining the app?

Reply:
Nope, not yet.